web-client/src/views/FileDocument/ExternalConsolidatedGroupCards.tsx

```diff
@@ -0,0 +1,71 @@
+import { connect } from '@cerebral/react';
+import { state } from 'cerebral';
+import React from 'react';
+
+export const ExternalConsolidatedGroupCards = connect(
+  {
+    externalConsolidatedCaseGroupHelper:
+      state.externalConsolidatedCaseGroupHelper,
+  },
+  function ExternalConsolidatedGroupCards({
+    externalConsolidatedCaseGroupHelper,
+  }) {
+    return (
+      <div className="grid-row grid-gap">
+        <div className="tablet:grid-col-6 margin-bottom-4">
+          <div className="card height-full margin-bottom-0">
+            <div className="content-wrapper">
+              <h3 className="underlined">
+                Case(s) The Document(s) Will Be Filed In
+              </h3>
+              <div className="grid-row grid-gap">
+                <div className="tablet:grid-col-12 margin-bottom-1">
+                  <div className="tablet:margin-bottom-0 margin-bottom-205">
+                    <label className="usa-label" htmlFor="primary-filing">
```

Comment:
Typically, I think `htmlFor` refers to the ID of the element the label is for. Is this label for some input or selection element?

Reply:
Probably shouldn't have been a label, changed to an h3!